Tween/Easings/EasingFunctionStore.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+
+namespace DUCK.Tween.Easings
+{
+	public class EasingFunctionStore
+	{
+		public static Dictionary<string, Func<float, float>> EasingFunctions { get; set; }
+		static EasingFunctionStore()
```

Comment:
Does this have to be a method? could it not be a backing field for the property?